xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererGL.cpp

```diff
@@ -191,14 +191,15 @@ COverlayTextureGL::COverlayTextureGL(CDVDOverlayImage* o, CRect& rSource)
     m_x = (0.5f * o->width + o->x) / o->source_width;
     m_y = (0.5f * o->height + o->y) / o->source_height;
 
-    int videoSourceH{static_cast<int>(rSource.Height())};
-    int videoSourceW{static_cast<int>(rSource.Width())};
+    float subRatio{static_cast<float>(o->source_width) / o->source_height};
+    float vidRatio{rSource.Width() / rSource.Height()};
 
-    if ((o->source_height == videoSourceH || videoSourceH % o->source_height == 0) &&
-        (o->source_width == videoSourceW || videoSourceW % o->source_width == 0))
+    // We always consider aligning 4/3 subtitles to the video,
+    // for example SD DVB subtitles (4/3) can be rendered on a fullhd video
+    bool isSubResSquare{subRatio == 1.25f || subRatio == 1.33f};
```

Comment:
Can you move this to a new/private method? Easier to read and to reuse. Also please in the docs of this new private method explain what/how the ratios 1.25 and 1.33 can be obtained

Reply:
done